.cicdtemplate/.codemagic/codemagic.yaml

```
@@ -52,13 +53,17 @@ workflows:
       events:
         - push
       branch_patterns:
-        - pattern: develop
+        - pattern: 'develop'
     scripts:
       - *detekt
       - *unit_test
       - name: Build APK for staging
         script: |
           ./gradlew assembleStagingDebug -PversionCode=$BUILD_NUMBER
+      - name: Generating release notes with the latest git commit
```

```suggestion
      - name: Generate release notes with the latest git commit
```
For consistency, please check the rest as well 🙏🏼